src/js/view/controls/templates/rightclick.js

```diff
@@ -25,7 +25,7 @@ const rightClickItem = (link = '', title = '', featured, showLogo) => {
 const infoOverlayItem = (videoInfoLocalization) => {
     return (
         `<li class="jw-reset jw-rightclick-item">` +
-            `<button class="jw-reset jw-rightclick-link jw-info-overlay-item">${videoInfoLocalization}</button>` +
+            `<a class="jw-reset jw-rightclick-link jw-info-overlay-item">${videoInfoLocalization}</a>` +
```

Comment:
Keep the button, add `type="button"`. We've fixed this issue before: https://github.com/jwplayer/jwplayer/pull/2320